llvm/include/llvm/Object/DXContainer.h

```diff
@@ -149,6 +150,38 @@ struct RootConstantView : RootParameterView {
   }
 };
 
+struct RootDescriptorView_V1_0 : RootParameterView {
```

Comment:
Whoops, this was meant to be in the last review batch. But if we have one struct we could do something like this?
```suggestion
struct RootDescriptorView : RootParameterView {
  llvm::Expected<dxbc::RST0::v1::RootDescriptor> read(uint32_t Version) {
    if (Version == 0)
      return readParameter<dxbc::RST0::v1::RootDescriptor, dxbc::RST0::v0::RootDescriptor>();
    return readParameter<dxbc::RST0::v1::RootDescriptor>();
  }
```